DirectProgramming/DPC++FPGA/Tutorials/Features/dsp_control/sample.json

```diff
@@ -0,0 +1,63 @@
+{
+  "guid": "77006CE8-2935-41C7-BA30-DF6EA07CD73A",
+  "name": "DSP Control",
+  "categories": ["Toolkit/oneAPI Direct Programming/DPC++ FPGA/Tutorials/Features"],
+  "description": "An Intel® FPGA tutorial demonstrating the DSP control feature",
+  "toolchain": ["dpcpp"],
+  "os": ["linux", "windows"],
+  "targetDevice": ["FPGA"],
+  "builder": ["ide", "cmake"],
+  "languages": [{"cpp":{}}],
+  "ciTests": {
+    "linux": [
+      {
+        "id": "fpga_emu",
+        "steps": [
+          "dpcpp --version",
+          "cd ../../..",
```

Comment:
Linux has no long path issues, no need to do this.  Please only do the directory messing about in Windows.